easybuild/tools/module_naming_scheme/lowercase_module_naming_scheme.py

```diff
@@ -0,0 +1,33 @@
+import os
+import re
+
+from easybuild.tools.module_naming_scheme.mns import ModuleNamingScheme
+from easybuild.tools.module_naming_scheme.utilities import det_full_ec_version
+
+
+class LowercaseModuleNamingScheme(ModuleNamingScheme):
```

Comment:
@lucamar Wouldn't it make sense to derive this from `EasyBuildMNS`, and rename it to `LowercaseEasyBuildMNS`?

Reply:
@boegel Sure, I can change it to a subclass of `EasyBuildMNS` and the filename `lowercase_mns.py`: I think that initially we just copied the structure of `categorized_mns.py` as a template